### ISSUE TITLE
get_sam_line() error

### ISSUE THREAD

**Issue by xkqqkx** [simon-anders/htseq#52]
When calling alignment.get_sam_line() I got following error:
```
python3/src/HTSeq/_HTSeq.pyx in HTSeq._HTSeq.SAM_Alignment.get_sam_line()

TypeError: sequence item 9: expected str instance, bytes found
```
Running HTSeq v 0.9.1 with Python 3.6.4
The offending line seems to be the return call in get_sam_line() where a string is formed by joining few items (seq and qualstr field are bytes and not str).

**Comment by iosonofabio** [comment]
Thanks, will have a look

**Comment by iosonofabio** [comment]
Confirmed and fixed it in b258a15, will be there in the new release. Please use the development version until then.
